contrib/offline/manage-offline-container-images.sh

```diff
@@ -118,11 +118,14 @@ function register_container_images() {
 		cp ${CURRENT_DIR}/registries.conf         ${TEMP_DIR}/registries.conf
 		sed -i s@"HOSTNAME"@"$(hostname)"@  ${TEMP_DIR}/registries.conf
 		sudo cp ${TEMP_DIR}/registries.conf   /etc/containers/registries.conf
+  elif [ "$(uname)" == "Darwin" ]; then
+    echo "This is a Mac, no configuration changes are required"
 	else
 		echo "runtime package(docker-ce, podman, nerctl, etc.) should be installed"
 		exit 1
 	fi
 
+	rm -rf ${IMAGE_DIR}
```

Comment:
Could you explain why you need to add this line?

Reply:
> Could you explain why you need to add this line?

For the convenience of testing. Let me delete this line of code.